chapters/hpmor-chapter-002.tex

```diff
@@ -1,7 +1,7 @@
 \chapter{Everything I Believe Is False}
 
 \begin{chapterOpeningAuthorNote}
-\#include "stddisclaimer.h"
+\#include “stddisclaimer.h”
```

Comment:
These ones should stay straight! I didn't annotate these ones :)

Reply:
ups, reg exp used ;-)

Reply:
and uncommented the whole line, since this one is quite strange and would result in problems here: https://github.com/rrthomas/hpmor/pull/69